test/helpers/password.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 // The password used for integration tests:
-let password = 'CozyTest$1'
+let password = process.env.COZY_PASSPHRASE || 'CozyTest_1'
```

Comment:
Don't think too much about this. I think this module will disappear. We will use a token generated by the stack, not the passphrase, in tests.